docs/database-functionalities/streams/implement-transformation-module.md

```diff
@@ -61,6 +61,7 @@ following:
 ```python
 import mgp
 
+@mgp.transformation
```

Comment:
I would move the text for adding the decorator first because the last paragraph goes:
`Last, but not least we have to mark our function as a transformation `
doesn't work as well with these changes. We are building the code snippet incrementally.

Reply:
Yep, I haven't read that part carefully. I'll go through it again!